2015-11-12 23:43:17,901 ERROR [main] o.l.g.c.GeogigCLI [GeogigCLI.java:408] An unhandled error occurred: Service  [RUNNING] has already been started. See the log for more details.
java.lang.IllegalStateException: Service  [RUNNING] has already been started
	at com.google.common.util.concurrent.AbstractService.startAsync(AbstractService.java:183) ~[guava-17.0.jar:na]
	at com.google.common.util.concurrent.AbstractScheduledService.startAsync(AbstractScheduledService.java:346) ~[guava-17.0.jar:na]
	at org.locationtech.geogig.metrics.RepositoryDecorator$1.opened(RepositoryDecorator.java:36) ~[geogig-metrics-1.0-beta1.jar:1.0-beta1]
	at org.locationtech.geogig.repository.Repository.open(Repository.java:112) ~[geogig-core-1.0-beta1.jar:1.0-beta1]
	at org.locationtech.geogig.api.porcelain.InitOp.callInternal(InitOp.java:227) ~[geogig-core-1.0-beta1.jar:1.0-beta1]
	at org.locationtech.geogig.api.porcelain.InitOp._call(InitOp.java:127) ~[geogig-core-1.0-beta1.jar:1.0-beta1]
	at org.locationtech.geogig.api.porcelain.InitOp._call(InitOp.java:64) ~[geogig-core-1.0-beta1.jar:1.0-beta1]
	at org.locationtech.geogig.api.AbstractGeoGigOp.call(AbstractGeoGigOp.java:125) ~[geogig-core-1.0-beta1.jar:1.0-beta1]
	at org.locationtech.geogig.cli.porcelain.Init.runInternal(Init.java:87) ~[geogig-cli-1.0-beta1.jar:1.0-beta1]
	at org.locationtech.geogig.cli.AbstractCommand.run(AbstractCommand.java:60) ~[geogig-cli-1.0-beta1.jar:1.0-beta1]
	at org.locationtech.geogig.cli.GeogigCLI.executeInternal(GeogigCLI.java:521) [geogig-cli-1.0-beta1.jar:1.0-beta1]
	at org.locationtech.geogig.cli.GeogigCLI.execute(GeogigCLI.java:379) [geogig-cli-1.0-beta1.jar:1.0-beta1]
	at org.locationtech.geogig.cli.GeogigCLI.main(GeogigCLI.java:324) [geogig-cli-1.0-beta1.jar:1.0-beta1]
2015-11-13 00:35:25,166 ERROR [main] o.l.g.c.GeogigCLI [GeogigCLI.java:399] null
org.apache.commons.dbcp.SQLNestedException: Cannot create PoolableConnectionFactory (FATAL: authentification par mot de passe ��chou��e pour l'utilisateur  �� postgres ��)
	at org.apache.commons.dbcp.BasicDataSource.createPoolableConnectionFactory(BasicDataSource.java:1549) ~[commons-dbcp-1.4.jar:1.4]
	at org.apache.commons.dbcp.BasicDataSource.createDataSource(BasicDataSource.java:1388) ~[commons-dbcp-1.4.jar:1.4]
	at org.apache.commons.dbcp.BasicDataSource.getConnection(BasicDataSource.java:1044) ~[commons-dbcp-1.4.jar:1.4]
	at org.geotools.data.jdbc.datasource.AbstractManageableDataSource.getConnection(AbstractManageableDataSource.java:48) ~[gt-jdbc-11.2.jar:na]
	at org.locationtech.geogig.geotools.cli.porcelain.AbstractPGCommand.getDataStore(AbstractPGCommand.java:78) ~[geogig-geotools-1.0-beta1.jar:1.0-beta1]
	at org.locationtech.geogig.geotools.cli.porcelain.PGImport.runInternal(PGImport.java:76) ~[geogig-geotools-1.0-beta1.jar:1.0-beta1]
	at org.locationtech.geogig.cli.AbstractCommand.run(AbstractCommand.java:60) ~[geogig-cli-1.0-beta1.jar:1.0-beta1]
	at org.locationtech.geogig.cli.GeogigCLI.executeInternal(GeogigCLI.java:521) [geogig-cli-1.0-beta1.jar:1.0-beta1]
	at org.locationtech.geogig.cli.GeogigCLI.execute(GeogigCLI.java:379) [geogig-cli-1.0-beta1.jar:1.0-beta1]
	at org.locationtech.geogig.cli.GeogigCLI.main(GeogigCLI.java:324) [geogig-cli-1.0-beta1.jar:1.0-beta1]
Caused by: org.postgresql.util.PSQLException: FATAL: authentification par mot de passe ��chou��e pour l'utilisateur  �� postgres ��
	at org.postgresql.core.v3.ConnectionFactoryImpl.doAuthentication(ConnectionFactoryImpl.java:291) ~[postgresql-8.4-701.jdbc3.jar:na]
	at org.postgresql.core.v3.ConnectionFactoryImpl.openConnectionImpl(ConnectionFactoryImpl.java:108) ~[postgresql-8.4-701.jdbc3.jar:na]
	at org.postgresql.core.ConnectionFactory.openConnection(ConnectionFactory.java:66) ~[postgresql-8.4-701.jdbc3.jar:na]
	at org.postgresql.jdbc2.AbstractJdbc2Connection.<init>(AbstractJdbc2Connection.java:125) ~[postgresql-8.4-701.jdbc3.jar:na]
	at org.postgresql.jdbc3.AbstractJdbc3Connection.<init>(AbstractJdbc3Connection.java:30) ~[postgresql-8.4-701.jdbc3.jar:na]
	at org.postgresql.jdbc3.Jdbc3Connection.<init>(Jdbc3Connection.java:24) ~[postgresql-8.4-701.jdbc3.jar:na]
	at org.postgresql.Driver.makeConnection(Driver.java:393) ~[postgresql-8.4-701.jdbc3.jar:na]
	at org.postgresql.Driver.connect(Driver.java:267) ~[postgresql-8.4-701.jdbc3.jar:na]
	at org.apache.commons.dbcp.DriverConnectionFactory.createConnection(DriverConnectionFactory.java:38) ~[commons-dbcp-1.4.jar:1.4]
	at org.apache.commons.dbcp.PoolableConnectionFactory.makeObject(PoolableConnectionFactory.java:582) ~[commons-dbcp-1.4.jar:1.4]
	at org.apache.commons.dbcp.BasicDataSource.validateConnectionFactory(BasicDataSource.java:1556) ~[commons-dbcp-1.4.jar:1.4]
	at org.apache.commons.dbcp.BasicDataSource.createPoolableConnectionFactory(BasicDataSource.java:1545) ~[commons-dbcp-1.4.jar:1.4]
	... 9 common frames omitted
2015-11-13 00:35:43,487 ERROR [main] o.l.g.c.GeogigCLI [GeogigCLI.java:399] null
org.apache.commons.dbcp.SQLNestedException: Cannot create PoolableConnectionFactory (FATAL: authentification par mot de passe ��chou��e pour l'utilisateur  �� fred ��)
	at org.apache.commons.dbcp.BasicDataSource.createPoolableConnectionFactory(BasicDataSource.java:1549) ~[commons-dbcp-1.4.jar:1.4]
	at org.apache.commons.dbcp.BasicDataSource.createDataSource(BasicDataSource.java:1388) ~[commons-dbcp-1.4.jar:1.4]
	at org.apache.commons.dbcp.BasicDataSource.getConnection(BasicDataSource.java:1044) ~[commons-dbcp-1.4.jar:1.4]
	at org.geotools.data.jdbc.datasource.AbstractManageableDataSource.getConnection(AbstractManageableDataSource.java:48) ~[gt-jdbc-11.2.jar:na]
	at org.locationtech.geogig.geotools.cli.porcelain.AbstractPGCommand.getDataStore(AbstractPGCommand.java:78) ~[geogig-geotools-1.0-beta1.jar:1.0-beta1]
	at org.locationtech.geogig.geotools.cli.porcelain.PGImport.runInternal(PGImport.java:76) ~[geogig-geotools-1.0-beta1.jar:1.0-beta1]
	at org.locationtech.geogig.cli.AbstractCommand.run(AbstractCommand.java:60) ~[geogig-cli-1.0-beta1.jar:1.0-beta1]
	at org.locationtech.geogig.cli.GeogigCLI.executeInternal(GeogigCLI.java:521) [geogig-cli-1.0-beta1.jar:1.0-beta1]
	at org.locationtech.geogig.cli.GeogigCLI.execute(GeogigCLI.java:379) [geogig-cli-1.0-beta1.jar:1.0-beta1]
	at org.locationtech.geogig.cli.GeogigCLI.main(GeogigCLI.java:324) [geogig-cli-1.0-beta1.jar:1.0-beta1]
Caused by: org.postgresql.util.PSQLException: FATAL: authentification par mot de passe ��chou��e pour l'utilisateur  �� fred ��
	at org.postgresql.core.v3.ConnectionFactoryImpl.doAuthentication(ConnectionFactoryImpl.java:291) ~[postgresql-8.4-701.jdbc3.jar:na]
	at org.postgresql.core.v3.ConnectionFactoryImpl.openConnectionImpl(ConnectionFactoryImpl.java:108) ~[postgresql-8.4-701.jdbc3.jar:na]
	at org.postgresql.core.ConnectionFactory.openConnection(ConnectionFactory.java:66) ~[postgresql-8.4-701.jdbc3.jar:na]
	at org.postgresql.jdbc2.AbstractJdbc2Connection.<init>(AbstractJdbc2Connection.java:125) ~[postgresql-8.4-701.jdbc3.jar:na]
	at org.postgresql.jdbc3.AbstractJdbc3Connection.<init>(AbstractJdbc3Connection.java:30) ~[postgresql-8.4-701.jdbc3.jar:na]
	at org.postgresql.jdbc3.Jdbc3Connection.<init>(Jdbc3Connection.java:24) ~[postgresql-8.4-701.jdbc3.jar:na]
	at org.postgresql.Driver.makeConnection(Driver.java:393) ~[postgresql-8.4-701.jdbc3.jar:na]
	at org.postgresql.Driver.connect(Driver.java:267) ~[postgresql-8.4-701.jdbc3.jar:na]
	at org.apache.commons.dbcp.DriverConnectionFactory.createConnection(DriverConnectionFactory.java:38) ~[commons-dbcp-1.4.jar:1.4]
	at org.apache.commons.dbcp.PoolableConnectionFactory.makeObject(PoolableConnectionFactory.java:582) ~[commons-dbcp-1.4.jar:1.4]
	at org.apache.commons.dbcp.BasicDataSource.validateConnectionFactory(BasicDataSource.java:1556) ~[commons-dbcp-1.4.jar:1.4]
	at org.apache.commons.dbcp.BasicDataSource.createPoolableConnectionFactory(BasicDataSource.java:1545) ~[commons-dbcp-1.4.jar:1.4]
	... 9 common frames omitted
2015-11-13 00:36:34,208 ERROR [main] o.l.g.c.GeogigCLI [GeogigCLI.java:399] null
org.locationtech.geogig.geotools.plumbing.GeoToolsOpException: null
	at org.locationtech.geogig.geotools.plumbing.ImportOp.checkPreconditions(ImportOp.java:332) ~[geogig-geotools-1.0-beta1.jar:1.0-beta1]
	at org.locationtech.geogig.geotools.plumbing.ImportOp._call(ImportOp.java:129) ~[geogig-geotools-1.0-beta1.jar:1.0-beta1]
	at org.locationtech.geogig.geotools.plumbing.ImportOp._call(ImportOp.java:72) ~[geogig-geotools-1.0-beta1.jar:1.0-beta1]
	at org.locationtech.geogig.api.AbstractGeoGigOp.call(AbstractGeoGigOp.java:125) ~[geogig-core-1.0-beta1.jar:1.0-beta1]
	at org.locationtech.geogig.geotools.cli.porcelain.PGImport.runInternal(PGImport.java:81) ~[geogig-geotools-1.0-beta1.jar:1.0-beta1]
	at org.locationtech.geogig.cli.AbstractCommand.run(AbstractCommand.java:60) ~[geogig-cli-1.0-beta1.jar:1.0-beta1]
	at org.locationtech.geogig.cli.GeogigCLI.executeInternal(GeogigCLI.java:521) [geogig-cli-1.0-beta1.jar:1.0-beta1]
	at org.locationtech.geogig.cli.GeogigCLI.execute(GeogigCLI.java:379) [geogig-cli-1.0-beta1.jar:1.0-beta1]
	at org.locationtech.geogig.cli.GeogigCLI.main(GeogigCLI.java:324) [geogig-cli-1.0-beta1.jar:1.0-beta1]
2015-11-13 00:38:43,332 ERROR [main] o.l.g.c.GeogigCLI [GeogigCLI.java:408] An unhandled error occurred: Service  [RUNNING] has already been started. See the log for more details.
java.lang.IllegalStateException: Service  [RUNNING] has already been started
	at com.google.common.util.concurrent.AbstractService.startAsync(AbstractService.java:183) ~[guava-17.0.jar:na]
	at com.google.common.util.concurrent.AbstractScheduledService.startAsync(AbstractScheduledService.java:346) ~[guava-17.0.jar:na]
	at org.locationtech.geogig.metrics.RepositoryDecorator$1.opened(RepositoryDecorator.java:36) ~[geogig-metrics-1.0-beta1.jar:1.0-beta1]
	at org.locationtech.geogig.repository.Repository.open(Repository.java:112) ~[geogig-core-1.0-beta1.jar:1.0-beta1]
	at org.locationtech.geogig.api.porcelain.InitOp.callInternal(InitOp.java:227) ~[geogig-core-1.0-beta1.jar:1.0-beta1]
	at org.locationtech.geogig.api.porcelain.InitOp._call(InitOp.java:127) ~[geogig-core-1.0-beta1.jar:1.0-beta1]
	at org.locationtech.geogig.api.porcelain.InitOp._call(InitOp.java:64) ~[geogig-core-1.0-beta1.jar:1.0-beta1]
	at org.locationtech.geogig.api.AbstractGeoGigOp.call(AbstractGeoGigOp.java:125) ~[geogig-core-1.0-beta1.jar:1.0-beta1]
	at org.locationtech.geogig.cli.porcelain.Init.runInternal(Init.java:87) ~[geogig-cli-1.0-beta1.jar:1.0-beta1]
	at org.locationtech.geogig.cli.AbstractCommand.run(AbstractCommand.java:60) ~[geogig-cli-1.0-beta1.jar:1.0-beta1]
	at org.locationtech.geogig.cli.GeogigCLI.executeInternal(GeogigCLI.java:521) [geogig-cli-1.0-beta1.jar:1.0-beta1]
	at org.locationtech.geogig.cli.GeogigCLI.execute(GeogigCLI.java:379) [geogig-cli-1.0-beta1.jar:1.0-beta1]
	at org.locationtech.geogig.cli.GeogigCLI.main(GeogigCLI.java:324) [geogig-cli-1.0-beta1.jar:1.0-beta1]
2015-11-13 01:48:37,795 ERROR [main] o.l.g.c.GeogigCLI [GeogigCLI.java:408] An unhandled error occurred: Service  [RUNNING] has already been started. See the log for more details.
java.lang.IllegalStateException: Service  [RUNNING] has already been started
	at com.google.common.util.concurrent.AbstractService.startAsync(AbstractService.java:183) ~[guava-17.0.jar:na]
	at com.google.common.util.concurrent.AbstractScheduledService.startAsync(AbstractScheduledService.java:346) ~[guava-17.0.jar:na]
	at org.locationtech.geogig.metrics.RepositoryDecorator$1.opened(RepositoryDecorator.java:36) ~[geogig-metrics-1.0-beta1.jar:1.0-beta1]
	at org.locationtech.geogig.repository.Repository.open(Repository.java:112) ~[geogig-core-1.0-beta1.jar:1.0-beta1]
	at org.locationtech.geogig.api.porcelain.InitOp.callInternal(InitOp.java:227) ~[geogig-core-1.0-beta1.jar:1.0-beta1]
	at org.locationtech.geogig.api.porcelain.InitOp._call(InitOp.java:127) ~[geogig-core-1.0-beta1.jar:1.0-beta1]
	at org.locationtech.geogig.api.porcelain.InitOp._call(InitOp.java:64) ~[geogig-core-1.0-beta1.jar:1.0-beta1]
	at org.locationtech.geogig.api.AbstractGeoGigOp.call(AbstractGeoGigOp.java:125) ~[geogig-core-1.0-beta1.jar:1.0-beta1]
	at org.locationtech.geogig.cli.porcelain.Init.runInternal(Init.java:87) ~[geogig-cli-1.0-beta1.jar:1.0-beta1]
	at org.locationtech.geogig.cli.AbstractCommand.run(AbstractCommand.java:60) ~[geogig-cli-1.0-beta1.jar:1.0-beta1]
	at org.locationtech.geogig.cli.GeogigCLI.executeInternal(GeogigCLI.java:521) [geogig-cli-1.0-beta1.jar:1.0-beta1]
	at org.locationtech.geogig.cli.GeogigCLI.execute(GeogigCLI.java:379) [geogig-cli-1.0-beta1.jar:1.0-beta1]
	at org.locationtech.geogig.cli.GeogigCLI.main(GeogigCLI.java:324) [geogig-cli-1.0-beta1.jar:1.0-beta1]
2015-11-13 01:50:00,822 ERROR [main] o.l.g.c.GeogigCLI [GeogigCLI.java:399] null
org.locationtech.geogig.api.porcelain.NothingToCommitException: Nothing to commit after 0000000000000000000000000000000000000000
	at org.locationtech.geogig.api.porcelain.CommitOp._call(CommitOp.java:315) ~[geogig-core-1.0-beta1.jar:1.0-beta1]
	at org.locationtech.geogig.api.porcelain.CommitOp._call(CommitOp.java:52) ~[geogig-core-1.0-beta1.jar:1.0-beta1]
	at org.locationtech.geogig.api.AbstractGeoGigOp.call(AbstractGeoGigOp.java:125) ~[geogig-core-1.0-beta1.jar:1.0-beta1]
	at org.locationtech.geogig.cli.porcelain.Commit.runInternal(Commit.java:110) ~[geogig-cli-1.0-beta1.jar:1.0-beta1]
	at org.locationtech.geogig.cli.AbstractCommand.run(AbstractCommand.java:60) ~[geogig-cli-1.0-beta1.jar:1.0-beta1]
	at org.locationtech.geogig.cli.GeogigCLI.executeInternal(GeogigCLI.java:521) [geogig-cli-1.0-beta1.jar:1.0-beta1]
	at org.locationtech.geogig.cli.GeogigCLI.execute(GeogigCLI.java:379) [geogig-cli-1.0-beta1.jar:1.0-beta1]
	at org.locationtech.geogig.cli.GeogigCLI.main(GeogigCLI.java:324) [geogig-cli-1.0-beta1.jar:1.0-beta1]
2015-11-13 01:53:34,876 ERROR [main] o.l.g.c.GeogigCLI [GeogigCLI.java:408] An unhandled error occurred: HEAD has no commits yet, branch cannot be created.. See the log for more details.
java.lang.IllegalArgumentException: HEAD has no commits yet, branch cannot be created.
	at com.google.common.base.Preconditions.checkArgument(Preconditions.java:125) ~[guava-17.0.jar:na]
	at org.locationtech.geogig.api.porcelain.BranchCreateOp.resolveOriginCommitId(BranchCreateOp.java:120) ~[geogig-core-1.0-beta1.jar:1.0-beta1]
	at org.locationtech.geogig.api.porcelain.BranchCreateOp._call(BranchCreateOp.java:103) ~[geogig-core-1.0-beta1.jar:1.0-beta1]
	at org.locationtech.geogig.api.porcelain.BranchCreateOp._call(BranchCreateOp.java:31) ~[geogig-core-1.0-beta1.jar:1.0-beta1]
	at org.locationtech.geogig.api.AbstractGeoGigOp.call(AbstractGeoGigOp.java:125) ~[geogig-core-1.0-beta1.jar:1.0-beta1]
	at org.locationtech.geogig.cli.porcelain.Branch.runInternal(Branch.java:165) ~[geogig-cli-1.0-beta1.jar:1.0-beta1]
	at org.locationtech.geogig.cli.AbstractCommand.run(AbstractCommand.java:60) ~[geogig-cli-1.0-beta1.jar:1.0-beta1]
	at org.locationtech.geogig.cli.GeogigCLI.executeInternal(GeogigCLI.java:521) [geogig-cli-1.0-beta1.jar:1.0-beta1]
	at org.locationtech.geogig.cli.GeogigCLI.execute(GeogigCLI.java:379) [geogig-cli-1.0-beta1.jar:1.0-beta1]
	at org.locationtech.geogig.cli.GeogigCLI.main(GeogigCLI.java:324) [geogig-cli-1.0-beta1.jar:1.0-beta1]
2015-11-13 01:53:39,471 ERROR [main] o.l.g.c.GeogigCLI [GeogigCLI.java:408] An unhandled error occurred: HEAD has no commits yet, branch cannot be created.. See the log for more details.
java.lang.IllegalArgumentException: HEAD has no commits yet, branch cannot be created.
	at com.google.common.base.Preconditions.checkArgument(Preconditions.java:125) ~[guava-17.0.jar:na]
	at org.locationtech.geogig.api.porcelain.BranchCreateOp.resolveOriginCommitId(BranchCreateOp.java:120) ~[geogig-core-1.0-beta1.jar:1.0-beta1]
	at org.locationtech.geogig.api.porcelain.BranchCreateOp._call(BranchCreateOp.java:103) ~[geogig-core-1.0-beta1.jar:1.0-beta1]
	at org.locationtech.geogig.api.porcelain.BranchCreateOp._call(BranchCreateOp.java:31) ~[geogig-core-1.0-beta1.jar:1.0-beta1]
	at org.locationtech.geogig.api.AbstractGeoGigOp.call(AbstractGeoGigOp.java:125) ~[geogig-core-1.0-beta1.jar:1.0-beta1]
	at org.locationtech.geogig.cli.porcelain.Branch.runInternal(Branch.java:165) ~[geogig-cli-1.0-beta1.jar:1.0-beta1]
	at org.locationtech.geogig.cli.AbstractCommand.run(AbstractCommand.java:60) ~[geogig-cli-1.0-beta1.jar:1.0-beta1]
	at org.locationtech.geogig.cli.GeogigCLI.executeInternal(GeogigCLI.java:521) [geogig-cli-1.0-beta1.jar:1.0-beta1]
	at org.locationtech.geogig.cli.GeogigCLI.execute(GeogigCLI.java:379) [geogig-cli-1.0-beta1.jar:1.0-beta1]
	at org.locationtech.geogig.cli.GeogigCLI.main(GeogigCLI.java:324) [geogig-cli-1.0-beta1.jar:1.0-beta1]
2015-11-13 01:53:44,013 ERROR [main] o.l.g.c.GeogigCLI [GeogigCLI.java:408] An unhandled error occurred: HEAD has no commits yet, branch cannot be created.. See the log for more details.
java.lang.IllegalArgumentException: HEAD has no commits yet, branch cannot be created.
	at com.google.common.base.Preconditions.checkArgument(Preconditions.java:125) ~[guava-17.0.jar:na]
	at org.locationtech.geogig.api.porcelain.BranchCreateOp.resolveOriginCommitId(BranchCreateOp.java:120) ~[geogig-core-1.0-beta1.jar:1.0-beta1]
	at org.locationtech.geogig.api.porcelain.BranchCreateOp._call(BranchCreateOp.java:103) ~[geogig-core-1.0-beta1.jar:1.0-beta1]
	at org.locationtech.geogig.api.porcelain.BranchCreateOp._call(BranchCreateOp.java:31) ~[geogig-core-1.0-beta1.jar:1.0-beta1]
	at org.locationtech.geogig.api.AbstractGeoGigOp.call(AbstractGeoGigOp.java:125) ~[geogig-core-1.0-beta1.jar:1.0-beta1]
	at org.locationtech.geogig.cli.porcelain.Branch.runInternal(Branch.java:165) ~[geogig-cli-1.0-beta1.jar:1.0-beta1]
	at org.locationtech.geogig.cli.AbstractCommand.run(AbstractCommand.java:60) ~[geogig-cli-1.0-beta1.jar:1.0-beta1]
	at org.locationtech.geogig.cli.GeogigCLI.executeInternal(GeogigCLI.java:521) [geogig-cli-1.0-beta1.jar:1.0-beta1]
	at org.locationtech.geogig.cli.GeogigCLI.execute(GeogigCLI.java:379) [geogig-cli-1.0-beta1.jar:1.0-beta1]
	at org.locationtech.geogig.cli.GeogigCLI.main(GeogigCLI.java:324) [geogig-cli-1.0-beta1.jar:1.0-beta1]
2015-11-13 01:53:48,541 ERROR [main] o.l.g.c.GeogigCLI [GeogigCLI.java:408] An unhandled error occurred: HEAD has no commits yet, branch cannot be created.. See the log for more details.
java.lang.IllegalArgumentException: HEAD has no commits yet, branch cannot be created.
	at com.google.common.base.Preconditions.checkArgument(Preconditions.java:125) ~[guava-17.0.jar:na]
	at org.locationtech.geogig.api.porcelain.BranchCreateOp.resolveOriginCommitId(BranchCreateOp.java:120) ~[geogig-core-1.0-beta1.jar:1.0-beta1]
	at org.locationtech.geogig.api.porcelain.BranchCreateOp._call(BranchCreateOp.java:103) ~[geogig-core-1.0-beta1.jar:1.0-beta1]
	at org.locationtech.geogig.api.porcelain.BranchCreateOp._call(BranchCreateOp.java:31) ~[geogig-core-1.0-beta1.jar:1.0-beta1]
	at org.locationtech.geogig.api.AbstractGeoGigOp.call(AbstractGeoGigOp.java:125) ~[geogig-core-1.0-beta1.jar:1.0-beta1]
	at org.locationtech.geogig.cli.porcelain.Branch.runInternal(Branch.java:165) ~[geogig-cli-1.0-beta1.jar:1.0-beta1]
	at org.locationtech.geogig.cli.AbstractCommand.run(AbstractCommand.java:60) ~[geogig-cli-1.0-beta1.jar:1.0-beta1]
	at org.locationtech.geogig.cli.GeogigCLI.executeInternal(GeogigCLI.java:521) [geogig-cli-1.0-beta1.jar:1.0-beta1]
	at org.locationtech.geogig.cli.GeogigCLI.execute(GeogigCLI.java:379) [geogig-cli-1.0-beta1.jar:1.0-beta1]
	at org.locationtech.geogig.cli.GeogigCLI.main(GeogigCLI.java:324) [geogig-cli-1.0-beta1.jar:1.0-beta1]
2015-11-13 01:53:53,229 ERROR [main] o.l.g.c.GeogigCLI [GeogigCLI.java:408] An unhandled error occurred: HEAD has no commits yet, branch cannot be created.. See the log for more details.
java.lang.IllegalArgumentException: HEAD has no commits yet, branch cannot be created.
	at com.google.common.base.Preconditions.checkArgument(Preconditions.java:125) ~[guava-17.0.jar:na]
	at org.locationtech.geogig.api.porcelain.BranchCreateOp.resolveOriginCommitId(BranchCreateOp.java:120) ~[geogig-core-1.0-beta1.jar:1.0-beta1]
	at org.locationtech.geogig.api.porcelain.BranchCreateOp._call(BranchCreateOp.java:103) ~[geogig-core-1.0-beta1.jar:1.0-beta1]
	at org.locationtech.geogig.api.porcelain.BranchCreateOp._call(BranchCreateOp.java:31) ~[geogig-core-1.0-beta1.jar:1.0-beta1]
	at org.locationtech.geogig.api.AbstractGeoGigOp.call(AbstractGeoGigOp.java:125) ~[geogig-core-1.0-beta1.jar:1.0-beta1]
	at org.locationtech.geogig.cli.porcelain.Branch.runInternal(Branch.java:165) ~[geogig-cli-1.0-beta1.jar:1.0-beta1]
	at org.locationtech.geogig.cli.AbstractCommand.run(AbstractCommand.java:60) ~[geogig-cli-1.0-beta1.jar:1.0-beta1]
	at org.locationtech.geogig.cli.GeogigCLI.executeInternal(GeogigCLI.java:521) [geogig-cli-1.0-beta1.jar:1.0-beta1]
	at org.locationtech.geogig.cli.GeogigCLI.execute(GeogigCLI.java:379) [geogig-cli-1.0-beta1.jar:1.0-beta1]
	at org.locationtech.geogig.cli.GeogigCLI.main(GeogigCLI.java:324) [geogig-cli-1.0-beta1.jar:1.0-beta1]
